pkg/paho-mqtt/include/mqtt.h

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright (C) 2019  Javier FILEIV <javier.fileiv@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ *
+ */
+/**
+ * @ingroup     pkg_paho-mqtt
+ * @{
+ *
+ * @file
+ * @brief       Network MQTT interface definitions
+ *
+ * @author      Javier FILEIV <javier.fileiv@gmail.com>
+ */
+#ifndef MQTT_H
+#define MQTT_H
+
+#include <mutex.h>
+#include <xtimer.h>
+#include <thread.h>
+#include <net/sock/tcp.h>
```

Comment:
You've missed those in your last commit.

Reply:
Ups... thanks :)